colossalai/fx/profiler/profiler.py

```diff
@@ -257,18 +331,27 @@ def f(*args: Tuple[Argument, ...], **kwargs: Dict[str, Any]) -> Any:
         # If there is an argument that this `call_module` is inplace, we should
         # still run the profiling but discard some results regarding `module`.
         inplace = getattr(module, 'inplace', False)
+
+        # calculate parameter size
+        param_size = 0
+        for param in module.parameters():
+            param_size += param.numel() * param.element_size()
```

Comment:
can use the `parameter_size` defined in `memory.py`

Reply:
okay

Reply:
fixed~